Assets/Tests/Runtime/Host/NetworkIdentityTests.cs

```diff
@@ -172,7 +172,7 @@ public void RemoveclientAuthorityNotSpawned()
         [Test]
         public void RemoveClientAuthorityOfOwner()
         {
-            serverObjectManager.AddCharacter(server.LocalPlayer, gameObject);
+            serverObjectManager.ReplaceCharacter(server.LocalPlayer, server.LocalClient, gameObject);
```

Comment:
```suggestion
            serverObjectManager.ReplaceCharacter(server.LocalPlayer, gameObject);
```

client removed in #774